apps/gateway/src/routes/v2/documents/get/get.handler.test.ts

```diff
@@ -18,7 +18,7 @@ describe('GET documents', () => {
   describe('unauthorized', () => {
     it('fails', async () => {
       const res = await app.request(
-        '/api/v1/projects/1/versions/asldkfjhsadl/documents/path/to/document',
+        '/api/v2/projects/1/versions/asldkfjhsadl/documents/path/to/document',
```

Comment:
why the change from v1 to v2?

Reply:
Before the implementation was in v1. So the test. Now I moved the implementation to v2, so the test